tests/test_core.py

```diff
@@ -101,14 +104,14 @@ def test_dagobah_add_tasks():
 
 
 @with_setup(blank_dagobah)
-def test_dagobah_delete_job():
+def test_dagobah_delete_job2():
     dagobah.add_job('test_job')
```

Comment:
I renamed these two because they have the same name as an existing test, but are slightly different in how they test. Not sure if one is better than the other. But pyflakes was bitching.